### PR TITLE
fixing the bug that breaks the site when a ward is selected

### PR DESCRIPTION
**Pull request by emre2038**
## Description
investigating and fixing the bug that breaks the site when a ward is selected

## Related Issue
https://trello.com/c/dSHDxoSs/864-site-breaks-when-i-select-a-ward

## How to test it locally
On sanef profile

* select johannesburg
* select a ward
* open the rich data view
* confirm that the ward data is displayed instead of johannesburg and there is no console error

## Screenshots
image

### Updated
* `src/js/setup/dataexplorer.js` to correct the variable name

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design matches the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

**Comment by emre2038** [comment]
[site breaks when I select a ward](https://trello.com/c/dSHDxoSs/864-site-breaks-when-i-select-a-ward)

**Comment by michaelglenister** [comment]
A local test shows this fixed the issue, ready to merge

**Review by michaelglenister — APPROVED** [review]
A local test shows this fixed the issue, ready to merge. Thanks

**Comment by adieyal** [comment]
Code looks good - please add a unit test

**Comment by emre2038** [comment]
added a unit test @adieyal

File: src/js/elements/menu.js [hunk 1]
```diff
@@ -5,22 +5,10 @@ const hideondeployClsName = 'hideondeploy';
 const parentContainer = $(".data-mapper-content__list");
 let categoryTemplate = null;
 let subCategoryTemplate = null;
-let indicatorTemplate = null;
```
Comment (emre2038) [inline src/js/elements/menu.js]: @adieyal is there a reason you removed indicatorTemplate declaration? this let to the reference error

**Pull request merged**